reefer_monitoring/v1/README.md

```diff
@@ -0,0 +1,7 @@
+## Reefer Monitoring
+
+The DCSA Interface Standard for Reefer Monitoring is documented on [**DCSA Reefer Monitoring**](https://app.swaggerhub.com/apis/dcsaorg/DCSA_REEFER_OPERATIONAL) SwaggerHub.
+
+<a name="v100Beta1"></a>[Release v1.0.0-Beta-1 (1 November 2023)](https://app.swaggerhub.com/apis-docs/dcsaorg/DCSA_REEFER_OPERATIONAL/1.0.0-Beta-1)
```

Comment:
Normally I add the publication date here (once you press the "publish" button in SwaggerHub and the API is in readOnly state. Until then I use the "Unpublished" or similar text (up to you ;-) )

Reply:
changed to `unpublished`